acq4/devices/Device.py

```diff
@@ -56,7 +59,23 @@ def deviceInterface(self, win):
     def taskInterface(self, task):
         """Return a widget with a UI to put in the task rack"""
         return TaskGui(self, task)
-        
+
+    def defaultGeometryArgs(self):
+        """Return a dictionary of default arguments to be used when creating a 3D model for this device.
+        """
+        return {}
+
+    def getGeometries(self) -> list:
+        """Return a 3D model to be displayed in the 3D visualization window.
+        """
+        from acq4.modules.Visualize3D import create_geometry
+
+        defaults = self.defaultGeometryArgs()
+        if self._geometry:
+            args = {**self._geometry}
+            return create_geometry(defaults=defaults, **args)
```

Comment:
It would be nice if we could apply these defaults deeply. E.g. setting the rotation on the transform without overriding the entire transform.